tests/shellcheck/run_shellcheck.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# If $top_srcdir has not been set by automake, detect it
+if [ -z "$top_srcdir" ]; then
+    top_srcdir="$(realpath "$(dirname "$0")/../..")"
+fi
+
+source "${top_srcdir}/tests/shellcheck/ignores.sh"
+logfile="${top_srcdir}/tests/shellcheck/shellcheck.log"
```

Comment:
I have noticed that we create the `run_shellcheck.log` and `shellcheck.log` files with pretty much the same content. What is the reasoning behind it?

Reply:
I had no idea the output is saved to file. If I can remove the logging, that makes the script more simple.